packages/amplify-category-function/provider-utils/awscloudformation/function-template-dir/trigger-dynamodb.js

```diff
@@ -0,0 +1,9 @@
+exports.handler = function (event, context) { //eslint-disable-line
+  console.log(JSON.stringify(event, null, 2));
+  event.Records.forEach((record) => {
+    console.log(record.eventID);
+    console.log(record.eventName);
+    console.log('DynamoDB Record: %j', record.dynamodb);
+  });
+  context.done(null, 'Successfully processed DynamoDB record'); // SUCCESS with message
+};
```

Comment:
why not use the callback model since this isn't an async function? https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-handler.html

Reply:
absolutely, actually originally just copied this template from https://github.com/aws-amplify/amplify-cli/blob/08c0c706bce7fd5996ce7c782512f694c1ff0455/packages/amplify-category-storage/provider-utils/awscloudformation/triggers/dynamoDB/index.js#L1-L10

Reply:
Not suggesting you change it. Just curious. If that's the template used then follow that.

Reply:
it is a default lambda code template for @model trigger, thanks for bringing this up, it makes sense refining both templates. I mean, for now storage category trigger does not utilize function-category implementation, while actually having a very same logic for triggers, if this gets accepted, will be great to have another PR updating that utilizes function-category implementation in storage flow.